lib/Process/Image.pm

```diff
@@ -444,7 +444,11 @@ sub _process_output {
             $self->_add_step(["$Process::Globals::pamthreshold"]);
             push @args, "-g4";
         } else {
-            push @args, '-packbits';
+            unless ( $self->_is_grayscale($self->source->{metadata}) ) {
+                push @args, '-color';
+                push @args, '-truecolor';
+            }
+            push @args, '-flat';
```

Comment:
Is this short for `-flate`? I don't see a `-flat` option for pamtotiff.

Reply:
Yes, it's supposed to be flate.